doc/extra-large-conference/prosody.cfg.lua.visitor.template

```diff
@@ -0,0 +1,124 @@
+---------- Server-wide settings ----------
+component_interface = '0.0.0.0';
```

Comment:
why not localhost?

Reply:
Oh, this can be even disabled ... I will drop it